### ISSUE TITLE
Possibly fallback to no multisampling in glium examples if multisampling configuration fails?

### ISSUE THREAD

**Issue by daboross** [PistonDevelopers/conrod#1138]
Support for multisampling is currently broken on Wayland with X11 display mode, at least for me: https://github.com/glium/glium/issues/1677.

If it would be feasible to try starting the apps with multisampling first, and then start them without it if that fails that would be nice. I'll implement this in my conrod apps, but it would be nice to have in example apps too.

**Comment by mitchmindtree** [comment]
Yeah I like this idea @daboross :+1: It might add a bit of boilerplate but would probably be worth it considering how often the issue crops up for different hardware/platforms.
